pallets/collator-selection/src/tests.rs

```diff
@@ -61,6 +63,79 @@ fn it_should_set_invulnerables() {
 	});
 }
 
+#[test]
+fn add_invulnerable_works() {
+	new_test_ext().execute_with(|| {
```

Comment:
Nice to have a bound test for invulnerables (which leads to the error `TooManyInvulnerables`).